elephant/spike_train_generation.py

```diff
@@ -775,16 +833,15 @@ def single_interaction_process(
     # Generate the array of times for coincident events in SIP, not closer than
     # min_delay. The array is generated as a pq.Quantity.
     if coincidences == 'deterministic':
-        n_coincidences = int(
-            ((t_stop - t_start) * rate_c).rescale(pq.dimensionless))
+        n_coincidences = int(round(
```

Comment:
why did you add round?

Reply:
because if in average I expect 80.99 coincidences. It is more normal to use 81 than 80.

Reply:
While it works for the scenario you encountered, does it makes sense to round for all possible cases in the Universe of how this function can be called?

Reply:
Yes, since we want the closest approximation to the average coincidence count.